tests/remote_config/test_remote_configuration.py

```diff
@@ -310,7 +309,7 @@ def test_tracer_extra_services(self):
             if data["path"] == "/v0.7/config":
                 client_tracer = data["request"]["content"]["client"]["client_tracer"]
                 if "extra_services" in client_tracer:
-                    extra_services.append(client_tracer["extra_services"])
+                    extra_services.append(client_tracer["extra_services"] or [])
```

Comment:
This is the bug fix. If the services aren't set quickly enough `client_tracer["extra_services"]` can return `None`. The `None` throw an error when we later try and filter the list.